### ISSUE TITLE
Wavebox Portable?

### ISSUE THREAD

**Issue by My-Name-Is-Adam** [wavebox/waveboxapp#1110]
Would be great for environments where I am unable to install Wavebox but still want to use it... ie. My workplace.

Is this possible? 💯 

Thank you.

**Comment by Thomas101** [comment]
Not at the moment, but we have got some code running here that allows you to set the profile directory. With a little bit of playing around, you should be able to pre-configure this and have it run locally on a USB stick or similar.

I'll post up with some additional info when the release that supports this goes out :)

**Comment by AMDphreak** [comment]
Yeah it would be nice to be able to use personal subscription at work. Nobody I know at work knows about this app, and we have a review process, so they would have to purchase a duplicate license for one person who already has a license. I just found this app today and will be testing it out and likely getting a personal license.
Any thoughts on the legality or practicality of using personal license for a lone user at a workplace?
